lib/inspec/runner.rb

```diff
@@ -182,58 +182,10 @@ def register_rules(ctx)
 
     private
 
-    def block_source_info(block)
```

Comment:
Minor correction on my original comment: I move this to the rspec_runner not the rspec formatter.

I'm not sure if that difference influences your comment.  My thinking was that the only callsite of this function seems to be called in check_to_example (did I miss one) whose purpose seems tied precisely to the rspec_runner, so I moved both functions there.